run_model-dep-full_limits_per_channel.sh

```diff
@@ -113,20 +105,28 @@ if [[ $MODE == "initial" ]]; then
     ############
     # morphing
     ############
-    # morph_parallel.py --output ${defaultdir}/datacards \
-    #     --analysis ${analysis} \
-    #     --eras $ERA \
-    #     --category_list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/by_channel/sm_neuralnet_categories_$CHANNEL.txt \
-    #     --variable nnscore \
-    #     --sm \
-    #     --parallel 10 2>&1 | tee -a ${defaultdir}/logs/morph_sm_log.txt
+    morph_parallel.py --output ${defaultdir}/datacards \
+        --analysis ${analysis} \
+        --sub-analysis ${sub_analysis} \
+        --categorization ${categorization} \
+        --sm-like-hists ${sm_like_hists} \
+        --sm-gg-fractions ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/data/higgs_pt_reweighting_fullRun2_v2.root \
+        --eras 2016,2017,2018 \
+        --category_list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/by_channel/sm_neuralnet_categories_$CHANNEL.txt \
+        --variable nnscore \
+        --sm \
+        --parallel 10 2>&1 | tee -a ${defaultdir}/logs/morph_sm_log.txt
 
     morph_parallel.py --output ${defaultdir}/datacards \
         --analysis ${analysis} \
-        --eras $ERA \
+        --sub-analysis ${sub_analysis} \
+        --categorization ${categorization} \
+        --sm-like-hists ${sm_like_hists} \
+        --sm-gg-fractions ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/data/higgs_pt_reweighting_fullRun2_v2.root \
+        --eras 2016,2017,2018 \
         --category_list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/by_channel/mssm_signal_categories_$CHANNEL.txt \
```

Comment:
Are these the correct categories or the categories without the m_sv cut?

Reply:
This script is used explicitly with ML categories for some reason (maybe @harrypuuter created it?)

The channel-wise lists are created accordingly as far as I understand:

https://github.com/ArturAkh/MSSMvsSMRun2Legacy/blob/master/input/by_channel/mssm_signal_categories_mt.txt

Reply:
these are only the categories with the m_sv cut